FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+MessageSend.swift

```diff
@@ -28,87 +28,27 @@ extension ComposeViewController {
         // https://github.com/FlowCrypt/flowcrypt-ios/issues/291
         try await Task.sleep(nanoseconds: 100 * 1_000_000) // 100ms
 
-        let sendableMsg = try await self.composeMessageService.validateAndProduceSendableMsg(
-            senderEmail: selectedFromEmail,
-            input: self.input,
-            contextToSend: self.contextToSend
+        let sendableMsg = try await composeMessageService.validateAndProduceSendableMsg(
+            input: input,
+            contextToSend: contextToSend
         )
         UIApplication.shared.isIdleTimerDisabled = true
-        try await composeMessageService.encryptAndSend(
+        let identifier = try await composeMessageService.encryptAndSend(
             message: sendableMsg,
             threadId: input.threadId
         )
-        handleSuccessfullySentMessage()
-    }
 
-    internal func requestMissingPassPhraseWithModal(for signingKey: Keypair, isDraft: Bool = false) {
-        let alert = alertsFactory.makePassPhraseAlert(
-            onCancel: {
-                self.handle(error: ComposeMessageError.passPhraseRequired)
-            },
-            onCompletion: { [weak self] passPhrase in
-                guard let self = self else {
-                    return
-                }
-                Task<Void, Never> {
-                    do {
-                        let matched = try await self.composeMessageService.handlePassPhraseEntry(
-                            passPhrase,
-                            for: signingKey
-                        )
-                        if matched {
-                            if isDraft {
-                                self.saveDraftIfNeeded()
-                            } else {
-                                self.handleSendTap()
-                            }
-                        } else {
-                            self.handle(error: ComposeMessageError.passPhraseNoMatch)
-                        }
-                    } catch {
-                        self.handle(error: error)
-                    }
-                }
-            }
+        let messageIdentifier = MessageIdentifier(
+            draftId: input.type.info?.id,
+            threadId: Identifier(stringId: input.threadId),
+            messageId: identifier
         )
-        present(alert, animated: true, completion: nil)
-    }
-
-    internal func handle(error: Error) {
-        UIApplication.shared.isIdleTimerDisabled = false
-        hideSpinner()
-        navigationItem.rightBarButtonItem?.isEnabled = true
+        handleAction?(.sent(messageIdentifier))
```

Comment:
Is `handleAction` handler implemented somewhere? I couldn't find.

Reply:
`handleAction` is passed to `ComposeViewController` on init - https://github.com/FlowCrypt/flowcrypt-ios/blob/feature/issue-1813-drafts/FlowCrypt/Controllers/Compose/ComposeViewController.swift#L158

So view controller, from which `ComposeViewController` was opened, can handle actions like sending message or creating new draft.